XenAdmin/Wizards/NewPolicyWizard/NewPolicySnapshotTypePageSpecific.cs

```diff
@@ -166,6 +166,15 @@ public policy_backup_type BackupType
 
         private void RefreshTab(IVMPolicy policy)
         {
+            /* when a policy does not have any VMs, irrespective of
+             * the snapshot type, enable Quiesce 
+             */
+
+            if (policy.VMs.Count == 0)
+                quiesceCheckBox.Enabled = true;
+            else
+                quiesceCheckBox.Enabled = false;
+
             switch (policy.policy_type)
```

Comment:
I'm a bit confused with this method. The comment in line 169 is inconsistent with what's happening in this switch block.
If quiesce for 0 VMs shoud be enabled irrespective of the snapshot type, then you may want to move the check after the switch block, because quiesceCheckBox.Enabled will be overwritten in there.
On the other hand, if quiesce is only relevant for policy_backup_type.snapshot, you may want to move the check into this particular case and remove the comment.
What exactly is the intention here?